soaks/bin/analyze_experiment

```diff
@@ -55,23 +57,29 @@ for exp in bytes_written.experiment.unique():
     # samples actually have the same mean -- are from the same distribution --
     # and so there's some statistically interesting difference between the two
     # samples. For our purposes here that implies that performance has changed.
-    res = scipy.stats.ttest_ind_from_stats(baseline_mean,
+    mean_res = scipy.stats.ttest_ind_from_stats(baseline_mean,
                                            baseline_stdev,
                                            len(baseline),
                                            comparison_mean,
                                            comparison_stdev,
                                            len(comparison),
                                            equal_var=False)
+
+    stdev_res  = scipy.stats.bartlett(baseline.throughput.array, comparison.throughput.array)
+
     ttest_results.append({'experiment': exp,
-                          'Δ mean': diff.mean(),
-                          'Δ mean %': percent_change,
+                          'Δ mean': mean_diff.mean(),
+                          'Δ mean %': mean_percent_change,
+                          'ttest p-value': mean_res.pvalue,
+                          'Δ stdev': stdev_diff,
+                          'Δ stdev %': stdev_percent_change,
+                          'bartlett p-value': stdev_res.pvalue,
```

Comment:
So, if we display the Bartlett (or whatever non-normal insensitive variant we end up with) what are we telling our users about the data set? Reporting the t-test p-value feels kinda iffy to me, but that's something related to configuration users pass. I think as presented we'd need the users to be kinda clued in about what equal variances mean, how to interpret a p-value and what to do when the variances don't match. 

That's not necessarily a bad thing, something to consider. I'd, personally, drop this column from all but the most detailed table if we want to keep it. I do wonder, that said, if we might not consider inequal variance as a signal for erraticism, REF #10923

Reply:
These are good points. What would you think of, rather than adding the bartlett p-value as a column here, we instead just put `n/a` (or something similar) in the `Δ stdev` column if it isn't a statistically significant difference (e.g. `p-value < 0.05`) ? We could even apply this same rule to `Δ mean` and only report it if it is a statistically significant difference.

Reply:
I think I've slowly worked myself around to the idea of having a fully separate sub-comment or analysis workflow that prints the wonky data we need to develop the soaks, separate from the interpretive tables the users need. Personally I've started to chafe against the need to keep the diagnostic comment explicable for users while also wanting more reported data for the kind of work we're doing on the back side of this. [For instance](https://github.com/vectordotdev/vector/pull/11013#issuecomment-1020688452)